extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -353,14 +431,24 @@ export class SystemLayer {
    */
   onMuteAudioClick_() {
     this.storeService_.dispatch(Action.TOGGLE_MUTED, true);
+    const hideTimeout = 1500;
+    this.vsync_.mutate(() => {
+      this.getShadowRoot().setAttribute('messagedisplay', 'show');
+      this.hideAfterTimeout_(hideTimeout);
+    });
   }
-
+  
   /**
    * Handles click events on the unmute button.
    * @private
    */
   onUnmuteAudioClick_() {
     this.storeService_.dispatch(Action.TOGGLE_MUTED, false);
+    const hideTimeout = 1500;
```

Comment:
I wonder if the showing/hiding/timeout can just be done with CSS transitions?

Reply:
I'll look into that! I did just try to do what I saw with the hiding of the hint overlay in amp-story-hint.js. Is it redundant with the opacity changes I did with the CSS? Thanks for all the feedback by the way!